docs/reference/write-result-classes.txt

```diff
@@ -29,15 +29,10 @@ Methods
 .. toctree::
    :titlesonly:
 
-   /reference/method/MongoDBBulkWriteResult-getDeletedCount
-   /reference/method/MongoDBBulkWriteResult-getInsertedCount
-   /reference/method/MongoDBBulkWriteResult-getInsertedIds
-   /reference/method/MongoDBBulkWriteResult-getMatchedCount
-   /reference/method/MongoDBBulkWriteResult-getModifiedCount
-   /reference/method/MongoDBBulkWriteResult-getUpsertedCount
-   /reference/method/MongoDBBulkWriteResult-getUpsertedIds
-   /reference/method/MongoDBBulkWriteResult-isAcknowledged
+   /reference/class/MongoDBBulkWriteResult
 
+:ref:`BulkWriteResult Methods <php-bulk-write-result>`
```

Comment:
What is the reasoning for eliminating the list of linked methods from `write-result-classes.txt` and `result-classes.txt`? The original point of these pages were to avoid cluttering the navigation.

So instead of a concrete landing page for each result class (like we have with `MongoDBClient.txt`), we consolidated all of the content on these two pages.

I see this PR adds concrete landing pages for the result classes, but the only purpose seems to be including the list of methods. The "Description" headings that describe each class are still _here_, which is inconsistent with other class pages like `MongoDBClient.txt`.

I'd suggest either of the following:

 * Move "Definition" (the only remaining content on these pages) to the individual result class pages, and eliminate `write-result-classes.txt` and `result-classes.txt` altogether. In this case, we can lump all of the classes under a single "Result Classes" heading (no real need to differentiate write results).
 * Move the list of methods back to `write-result-classes.txt` and `result-classes.txt` and eliminate the need for individual pages for these classes (only keeping their methods).

Reply:
Okay, fixed according to your first suggestion (moved the "Definition" sections to the individual class pages)